index.html

```diff
@@ -181,6 +181,11 @@ <h4>Alexander Peschel</h4>
             </div>
 
             <a href="https://github.com/meetalva/alva/graphs/contributors" target="_blank">and many more contributors on Github!</a>
+
+            <div class="social">
+              <a class="social-item twitter" href="https://twitter.com/home?status=Alva%20is%20a%20radically%20new%20design%20tool%20that%20enables%20cross-functional%20teams%20to%20design%20digital%20products.%20https%3A//meetalva.github.io/" target="_blank">Tweet about Alva</a>
```

Comment:
We should use meetalva.io as a domain, not meetalva.github.io

Reply:
And we should use the new text (from the hero: Meet Alva, 
a radically new digital design tool built for cross-functional product teams.)